maker-core/cvc.md

```diff
@@ -1,41 +1,44 @@
 # Constitutional Voter Committees (CVCs)
 
-Constitutional Voter Committees (CVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Constitutional Delegates to direct the evolution of the Maker Protocol.
+## Overview
 
-{% hint style="warning" %}
```

Comment:
Keep the warning but change to: {% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}